Library/Formula/weechat.rb

```diff
@@ -30,12 +34,13 @@ def install
 
     args = std_cmake_args
 
-    args << "-DENABLE_LUA=OFF"    if build.without? "lua"
-    args << "-DENABLE_PERL=OFF"   if build.without? "perl"
-    args << "-DENABLE_RUBY=OFF"   if build.without? "ruby"
-    args << "-DENABLE_ASPELL=OFF" if build.without? "aspell"
-    args << "-DENABLE_GUILE=OFF"  if build.without? "guile"
-    args << "-DENABLE_PYTHON=OFF" if build.without? "python"
+    args << "-DENABLE_LUA=OFF"        if build.without? "lua"
```

Comment:
Ok, so this is because I didn't canceled the added spacing: at basis, I added them because I added a line `args << "-DENABLE_JAVASCRIPT=OFF"        if build.without? "javascript"`. I had to remove this one because there is an issue to build weechat with JS for now (I can't achieve to build with homebrew's packaged in 3.x, the simplest way will probably be to wait for weechat 1.3 and support for 4.3 and more).
Is this a real problem to keep those spaces while Javascript isn't back?